terraform/modules/spack_gitlab/modules/spackbot_personal_access_token/main.tf

```diff
@@ -0,0 +1,33 @@
+locals {
+  # GitLab no longer allows tokens to be created with an indefinite expiration date,
+  # so we set a long expiration date and a warning date to remind us to update it.
+  token_expiration_date = "2026-04-25"
```

Comment:
Just to be clear, this means that all PATs created with this module expire on the same date?

Reply:
Yes. That was already the case, but the date was replicated for each token. Now, all we need to do is update this one string to renew every token.